src/main/scala/com/softwaremill/realworld/profiles/ProfileDto.scala

```diff
@@ -0,0 +1,18 @@
+package com.softwaremill.realworld.profiles
```

Comment:
I think the naming convention is that if the file doesn't correspond to a class it should be lower-case

also - do we follow the `*Dto` naming convention elsewhere in the project?

Reply:
Yes, I was inspired by `UserDto` file.
However, I decided to extract each class into its own dedicated file 👌